src/libraries/System.Threading.RateLimiting/ref/System.Threading.RateLimiting.cs

```diff
@@ -13,7 +13,7 @@ public sealed partial class ConcurrencyLimiter : System.Threading.RateLimiting.R
         protected override System.Threading.RateLimiting.RateLimitLease AcquireCore(int permitCount) { throw null; }
         protected override void Dispose(bool disposing) { }
         protected override System.Threading.Tasks.ValueTask DisposeAsyncCore() { throw null; }
-        public override int GetAvailablePermits() { throw null; }
+        public override System.Threading.RateLimiting.RateLimiterStatistics? GetStatistics() { throw null; }
```

Comment:
Do we need a breaking change doc or was this API added in .NET 7?

Reply:
It's a new API added in .NET 7: https://github.com/dotnet/runtime/pull/61788